src/pages/index.tsx

```diff
@@ -1,57 +1,91 @@
+import Header from '../components/header/';
+import Hero from '../components/hero';
+import AgeGroups from '../components/ageGroups/frontPageSection';
+
 const IndexPage = () => {
   const mockHeaderItems = [
     {
-      name: "Partio-Ohjelma",
+      name: 'Partio-Ohjelma',
       subMenu: [
         {
-          name: "Sudenpennut",
-          ingress: "7-9 vuotiaat",
-          url: "/sudenpennut",
+          name: 'Sudenpennut',
+          ingress: '7-9 vuotiaat',
+          url: '/sudenpennut',
         },
         {
-          name: "Seikkailijat",
-          ingress: "10-12 vuotiaat",
-          url: "/seikkailijat",
+          name: 'Seikkailijat',
+          ingress: '10-12 vuotiaat',
+          url: '/seikkailijat',
         },
         {
-          name: "Tarpojat",
-          ingress: "15-17 vuotiaat",
-          url: "/tarpojat",
+          name: 'Tarpojat',
+          ingress: '15-17 vuotiaat',
+          url: '/tarpojat',
         },
         {
-          name: "Samoajat",
-          ingress: "15-17 vuotiaat",
-          url: "/samoajat",
+          name: 'Samoajat',
+          ingress: '15-17 vuotiaat',
+          url: '/samoajat',
         },
         {
-          name: "Vaeltajat",
-          ingress: "18-22 vuotiaat",
-          url: "/vaeltajat",
+          name: 'Vaeltajat',
+          ingress: '18-22 vuotiaat',
+          url: '/vaeltajat',
         },
         {
-          name: "Perhepartio",
-          ingress: "Lorem ipsum dolor",
-          url: "/perhepartio",
+          name: 'Perhepartio',
+          ingress: 'Lorem ipsum dolor',
+          url: '/perhepartio',
         },
       ],
     },
     {
-      name: "Ajankohtaista",
+      name: 'Ajankohtaista',
+    },
+    {
+      name: 'Materiaalit',
     },
     {
-      name: "Kampanjat",
+      name: 'Partiokasvatus',
+    },
+  ];
+
+  const mockNews = [
+    {
+      title: 'Lorem ipsum dolor ajankohtaista heading',
+      ingress:
+        'Nulla fringilla eros vel ex eleifend accumsan. Phasellus porta massa eu neque viverra varius. Fusce id sapien tortor. Sed bibendum eleifend...',
+      date: '1.8.2021',
+      buttonText: 'Lue lisää',
     },
     {
-      name: "Materiaalit",
+      title: 'Lorem ipsum dolor ajankohtaista ',
+      ingress:
+        'Nulla fringilla eros vel ex eleifend accumsan. Phasellus porta massa eu neque viverra varius. Fusce id sapien tortor. Sed bibendum eleifend...',
+      date: '28.8.2021',
+      buttonText: 'Lue lisää',
     },
     {
-      name: "Partiokasvatus",
+      title: 'Lorem ipsum dolor ajankohtaista ',
+      ingress:
+        'Nulla fringilla eros vel ex eleifend accumsan. Phasellus porta massa eu neque viverra varius. Fusce id sapien tortor. Sed bibendum eleifend...',
+      date: '28.8.2021',
+      buttonText: 'Lue lisää',
     },
   ];
 
-  return <Header headerItems={mockHeaderItems} />;
+  return (
+    <div>
+      <Header headerItems={mockHeaderItems} />
```

Comment:
Wanna move the header to a layout file which could be then used in all pages?

The layout component could be something like this (in somewhere like `layouts/default.tsx`):

```tsx
interface LayoutProps {
  children: React.ReactNode;
}

const DefaultLayout = ({ children }: LayoutProps) => {
  return (
    <div>
      <Header headerItems={mockHeaderItems} />
      <div className="container mx-auto">{children}</div>
    </div>
  );
};
```

Then use is here like:
```jsx
<DefaultLayout>
    <Hero />
    <CurrentNewsHighlights currentNews={mockNews} />
    <AgeGroups />
</DefaultLayout>
```